kem/xwing/xwing.go

```diff
@@ -283,7 +285,9 @@ func (sk *PrivateKey) DecapsulateTo(ss, ct []byte) {
 	copy(ctx[:], ct[mlkem768.CiphertextSize:])
 
 	sk.m.DecapsulateTo(ssm[:], ctm)
-	x25519.Shared(&ssx, &sk.x, &ctx)
+	if !x25519.Shared(&ssx, &sk.x, &ctx) {
```

Comment:
For X-Wing we can ignore for now.

Reply:
It would be nice to have this ambiguity clarified in the specification :)
https://github.com/dconnolly/draft-connolly-cfrg-xwing-kem/issues/28

See also Armando's [comment above](https://github.com/cloudflare/circl/pull/541#discussion_r2006633757) for an alternative suggestion.

Reply:
The linter didn't like the empty branch, so I just moved the comment above.